### ISSUE TITLE
Cannot resolve schema due to "undefined" using RAML 0.8

### ISSUE THREAD

**Issue by Narkhip** [raml-org/raml-js-parser-2#200]
## Description 

The following API has some Schema validations errors. This is a very important scenario, since a lot of our Production RAMLs are failing due to Schemas issues like this.

[Infinite_Home_-_Client_-_Reference-v_latest.zip](https://github.com/raml-org/raml-js-parser-2/files/267432/Infinite_Home_-_Client_-_Reference-v_latest.zip)

**Comment by petrochenko-pavel-a** [comment]
Hm, this works fine for me. I see schema related warnings but checking schemas in [JSON schema validator](http://www.jsonschemavalidator.net) shows that schemas actually contains errors.

**Comment by sichvoge** [comment]
That might be already fixed with #194

**Comment by petrochenko-pavel-a** [comment]
Tested with latest release all that I see is warnings like (and random testing of schemas actually shows that they have some issues):

```
{ code: 7,
    message: 'Example does not conform to schema:Content is not valid according to schema:Reference could not be resolved: http://api.reference/link.schema http://api.reference/link.schema',
    path: 'api.raml',
    start: 50407,
    end: 50414,
    line: 1445,
    column: 20,
    range: [ [Object], [Object] ],
    isWarning: true } ]
```

Regards,
Pavel
